calculadora_do_cidadao/fields.py

```diff
@@ -13,7 +13,23 @@ def deserialize(cls, value: str) -> Decimal:
         return super().deserialize(value.replace(",", "."))
 
 
+class UsDecimalField(fields.DecimalField):
```

Comment:
How this differs from Rows's default `DecimalField`? I ask that because we might not need this new field if that's Rows's standard.

Reply:
Thought it was a default to add fields again over a local scope, such as Django models. Removing it!